### PR TITLE
let foreman_proxy_content role handle the whole upgrade 

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: pipelines/upgrade_pipeline.yml [hunk 1]
```diff
@@ -4,7 +4,9 @@
 - import_playbook: upgrade/03-install_server.yaml
 - import_playbook: upgrade/04-install_proxy.yaml
 - import_playbook: upgrade/05-server_to_intermediate.yaml
+  when: forklift_upgrade_version_start != forklift_upgrade_version_intermediate
```
Comment (ekohl) [inline pipelines/upgrade_pipeline.yml:7]: One issue is that in CI it doesn't actually run this playbook but rather all individual ones

Comment (evgeni) [reply pipelines/upgrade_pipeline.yml:7]: oh. right. because we are clever.

Comment (evgeni) [reply pipelines/upgrade_pipeline.yml:7]: how about now?

Comment (evgeni) [reply pipelines/upgrade_pipeline.yml]: (it wouldn't have worked anyways, as the vars are only loaded inside the playbook)

**Pull request merged**